system/jlib/jstats.cpp

```diff
@@ -1431,6 +1431,10 @@ StringBuffer & StatsScopeId::getScopeText(StringBuffer & out) const
         return out.append(ChannelScopePrefix).append(id);
     case SSTdfuworkunit:
         return out.append(DFUWorkunitScopePrefix).append(name);
+    case SSTsection:
+        return out.append(SectionScopePrefix).append(name);
+    case SSToperation:
+        return out.append(OperationScopePrefix).append(name);
```

Comment:
Note, all operation scopes are prefixed with >.  It wouldn't ever produce the scope "compile" but ">compile".  Please see comment regarding keeping SSTcompilestage.